docs/tutorials/unraid.md

```diff
@@ -90,3 +90,19 @@ If you would like to schedule a periodic scan of your library, use the [`searchC
 If you would like to set up RSS scans, use the [`rssCadence`](../basics/options.md#rsscadence) option.
 
 If you would like to automatically check for cross-seeds when a download finishes, learn how to [set up with automatic searches](../basics/daemon#set-up-automatic-searches-for-finished-downloads).
+
+## Auto-Restart
+
+`cross-seed` by default will stop the container if a configuration option is suspected to be bad. This includes bad config options for the torrent client.
+
+This can lead to issues where `cross-seed` may fail to start because of a temporary issue with the torrent client, like long start time, network issues, etc.
+After this happens, the container will stay **stopped** until you manually start it again, or another scheduled process turns it back on.
+
+Because of this, it is recommended to add the auto-restart parameter to your `Extra Parameters` section in Unraid.
+
+![screenshot-cross-seed-extra-param-config](../../static/img/unraid-extra-parameters-restart.png)
+
+:::tip
+You need to toggle from `Basic View` to `Advanced View` to see this option
+![screenshot-cross-seed-extra-param-config](../../static/img/toggle-advanced-view-unraid.png)
+:::
```

Comment:
The tip is not needed as it is mentioned in the previous suggestion, and if you look at the other images included in the unraid tutorial you will see they are hosted as an asset on gh's cdn.

Simply paste the image into a comment box here on the cross-seed repo and it will upload it, and you can grab the link there.